core/src/main/java/hudson/model/Item.java

```diff
@@ -139,8 +139,6 @@
      *
      * @param g
      *      The {@link ItemGroup} instance used as context to evaluate the relative name of this item
-     * @return
```

Comment:
I think you should keep this `@return` and delete the other one.

Reply:
Thanks Matt, can you explain why?

Reply:
It looked more descriptive. Is it out of date? I didn't really look too closely into the context.

Reply:
> Is it out of date?

Yeah, it referred to a parameter `p` which didn't exist (the method only takes `ItemGroup g`).

> It looked more descriptive.

This makes sense. Now that we have [articulated our rationale](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews), I can properly address this feedback by combining the more descriptive content from the first `@return` with the example from the second `@return`, correcting the outdated parameter reference in the process.